build.sbt

```diff
@@ -67,7 +67,10 @@ lazy val commonJsSettings = Seq(
   tlVersionIntroduced ++= List("2.12", "2.13").map(_ -> "2.1.0").toMap
 )
 
+lazy val NativeLink = Tags.Tag("native-link")
+Global / concurrentRestrictions += Tags.limit(NativeLink, 2)
 lazy val commonNativeSettings = Seq(
+  Test / nativeLink := (Test / nativeLink).tag(NativeLink).value,
```

Comment:
This should be upstreamed :)
- https://github.com/scala-native/scala-native/issues/2853